public/components/incontext_insight/generate_popover_body.tsx

```diff
@@ -98,6 +103,7 @@ export const GeneratePopoverBody: React.FC<{
               `Please provide your insight on this ${summaryType}.`
             );
           }
+          reportMetric(usageCollection, metricAppName, 'generated');
```

Comment:
As we record `generated` metric, i think the metric type could be METRIC_TYPE.COUNT

Reply:
ok, I 'll change it to count.